lib/template.html

```diff
@@ -110,8 +112,8 @@
 {% endif %}
 
 {% if page.url %}
```

Comment:
Agree it's incorrect as written. If `site.url` isn't set, the output may be a little silly. The relevant commit is https://github.com/jekyll/jekyll-seo-tag/commit/df09740757543e6f5a5cf59401c9c7ec7e67362f.

I believe that should be `site.url`, not `page.url`, which means we'd actually have a canonical URL.